util/deephash/deephash.go

```diff
@@ -436,30 +430,65 @@ func genTypeHasher(t reflect.Type) typeHasherFunc {
 		default:
 			return genHashStructFields(t)
 		}
+	case reflect.Map:
+		return func(h *hasher, v addressableValue) bool {
+			// TODO: Check for nil.
```

Comment:
who should do this? :)

seems worth doing now, though. What'll happen otherwise?

Reply:
Done.